pages/video-detail/video-detail.view.html

```diff
@@ -14,37 +14,57 @@
 			<div>
 				<player video="Video.video.data.video" poster="Video.video.data.poster"></player>
 
-				<!-- Only if desktop >>> -->
-				<user-card class="card-user-info-desktop" user="Video.user.data"></user-card>
-				<!--" <<<<<<< -->
+				<div class="video-detail-map">
+					<ng-map default-style="true" center="Video.mapMarker" ng-if="Video.mapMarker.length > 0">
+						<marker ng-if="Video.video.data.location" position="{{ Video.mapMarker }}"
+								title="{{ Video.video.data.locationName }}" centered="true"></marker>
+					</ng-map>
+					<div ng-if="Video.mapMarker.length === 0" class="no-map">
+						No location selected
+					</div>
+				</div>
+
 			</div>
 			<section class="card-video-info">
 				<div>
-					<h1 ng-bind="Video.video.data.title"></h1>
+					<h1>
+						{{ ::Video.video.data.title }}
+						<md-icon ng-if="Video.video.data.verified == true || Video.video.data.verified == 'true'">
```

Comment:
don't we really know if verified's typeof is boolean or string?

Reply:
Use to be boolean, but I preffer to be sure if something brakes.